src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/Toolbar.js

```diff
@@ -11,7 +14,41 @@ export default class Toolbar extends React.PureComponent {
         return (
             <header className={toolbarStyles.toolbar}>
                 <nav>
-                    {toolbarStore.items.map((item) => (<Item key={item.title} {...item} />))}
+                    <div className={toolbarStyles.pullLeft}>
```

Comment:
I really don't like this `pullLeft` kind of classes... What happens if you want to add more styling to this component at any time? Do you rename the class then?